scripts/draft-translate.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import click
+
+from googletrans import Translator  # https://github.com/ssut/py-googletrans
```

Comment:
I think it's unnecessary to note the project repo here. We can find it in PyPI already.

https://pypi.org/project/googletrans/